packages/squiggle-lang/src/rescript/Utility/E.res

```diff
@@ -152,13 +152,20 @@ module I = {
   let toString = Js.Int.toString
 }
 
+exception Assertion(string)
+
 /* R for Result */
 module R = {
   let result = Rationale.Result.result
   let id = e => e |> result(U.id, U.id)
   let fmap = Rationale.Result.fmap
   let bind = Rationale.Result.bind
   let toExn = Belt.Result.getExn
+  let assertOk = (message: string, x: result<'a, 'b>): 'a =>
+    switch x {
+    | Ok(r) => r
+    | Error(_) => raise(Assertion(message))
+    }
```

Comment:
I'm not certain about the feel of this, in such a way that an adjustment to the name could quiet me. The fact that it unpacks and returns the inner value makes it seem a lot like `E.R.toExn`, right?

Reply:
Yes! Was chatting to Ozzie about this earlier. I'll fix that soon